client/src/components/new-request/analyze-characters.vue

```diff
@@ -81,3 +83,8 @@ export default class AnalyzeCharacters extends Vue {
 }
 
 </script>
+
+<style scoped lang="sass">
+.padding-50-px
+  padding: 25px 50px 25px 50px !important
```

Comment:
This classname no longer makes sense.

How about just setting padding on the v-container, eg, `py-6 px-12` ? (I know these resolve to 24 48 24 48 but I think we should move to rems wherever possible.)

Reply:
that works for me, I was just trying to be precise per the mockups but if the built in classes are good enough then I'm all for it

Reply:
When the numbers are large enough, a couple of px here or there isn't noticeable.

I've also noticed that Tracey is a bit flexible with these things :)